pinot-controller/src/main/java/org/apache/pinot/controller/recommender/realtime/provisioning/MemoryEstimator.java

```diff
@@ -387,9 +375,13 @@ private int getAvgMultiValues() {
   private long getMemoryForInvertedIndex(long totalMemoryForConsumingSegment) {
     // TODO: better way to estimate inverted indexes memory utilization
     long totalInvertedIndexSizeBytes = 0;
-    if (!_invertedIndexColumns.isEmpty()) {
+    Map<String, IndexConfig> invertedConfig = StandardIndexes.inverted().getConfig(_tableConfig, _schema);
```

Comment:
this is a lot of code to check if invertedIndexColumns exist. SHould we put this into a util so it can be reused by others and not accidentally use the old way?

Reply:
Others never have to do this. What they do is to use `FieldIndexConfigs`. That is the source of trust and that is what all code should use.

In this case, we don't have that object and we are not actually interested in other indexes. Therefore I tried to do not create that object. But it doesn't make sense to call `getConfig` so many times, so I'm moving the code to another function and calling it only  once.